File: BOJ/BOJ_4344_평균은넘겠지.py
tc = int(input())
for _ in range(tc):
    data = list(map(int, input().split()))

    n = data[0]
    avg = sum(data[1:]) // n

    count = 0
    for i in range(1, n + 1):
        if data[i] > avg:
            count += 1

    print("{:.3f}%".format((count / n) * 100))

File: BOJ/BOJ_4344_평균은넘겠지.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Main {

	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int tc = Integer.parseInt(br.readLine());
		
		for (int i = 0; i < tc; i++) { 
			StringTokenizer st = new StringTokenizer(br.readLine());
			int N = Integer.parseInt(st.nextToken());
			int[] arr =  new int[N];
			for (int j = 0; j < N; j++) { // n
				arr[j] = Integer.parseInt(st.nextToken());
			}
			
			double avg = Arrays.stream(arr).sum() / N;
			
			int count = 0;
			for (int j = 0; j < N; j++) {
				if (arr[j] > avg) {
					count++;
				}
			}
			
			double result = count / (double)N * 100;
			System.out.printf("%.3f%%\n", result);
		}
		
	}

}
